src/auslib/util/comparison.py

```diff
@@ -15,8 +15,8 @@ def has_operator(value):
 
 
 def get_op(pattern):
-    # only alphanumeric characters means no operator
-    if re.match(r"\w+", pattern):
+    # only alphanumeric or glob characters means no operator
+    if re.match(r"[\w\*]+", pattern):
```

Comment:
I'm wondering if it might be cleaner to treat `*` as an operator, and return a function that implements the comparison, rather than special casing `*` down in `MozillaVersion`. What do you think?